packages/filecoin-actor-utils/data/actor-descriptors.json

```diff
@@ -0,0 +1,2913 @@
+{
```

Comment:
niiiice! One thing i'm noticing - there's no method `0` in these descriptors - maybe because its always a transfer? how to handle method 0?

Reply:
Good call, Send is a bit of an odd one, as it's not part of the actors themselves and reflection would be different. Also Lotus doesn't show any params or return values, probably unless the user supplied params manually. Added support for it here:
https://github.com/glifio/filecoin-descriptors/commit/d9ff75e23004d1eb87a54f50bbb285c26de18902